data/brands/tourism/hotel.json

```diff
@@ -18,6 +18,17 @@
     }
   },
   "items": [
+      {
+      "displayName": "The Leading Hotels of the World",
+      "locationSet": {"include": ["001"]},
+      "matchNames": ["lhw"],
+      "tags": {
+        "brand": "The Leading Hotels of the World",
+        "brand:wikidata": "Q834396",
+        "name": "The Leading Hotels of the World",
```

Comment:
No strong opinion on if this is added from me, it doesn't feel like a normal brand.But people have mapped it as such, eg: https://www.openstreetmap.org/way/734574023#map=19/37.44357/25.33006 https://www.openstreetmap.org/way/218656514#map=19/32.05685/34.76235

However I do have a strong opinion on the name, it shouldn't be here.
```suggestion
```